src/features/multichain/components/SignerSetupWarning/InconsistentSignerSetupWarning.tsx

```diff
@@ -37,14 +36,19 @@ export const InconsistentSignerSetupWarning = () => {
   const isMultichainSafe = useIsMultichainSafe()
   const safeAddress = useSafeAddress()
   const currentChain = useCurrentChain()
+  const currency = useAppSelector(selectCurrency)
   const undeployedSafes = useAppSelector(selectUndeployedSafes)
   const { allMultiChainSafes } = useAllSafesGrouped()
 
   const multiChainGroupSafes = useMemo(
     () => allMultiChainSafes?.find((account) => sameAddress(safeAddress, account.safes[0].address))?.safes ?? [],
     [allMultiChainSafes, safeAddress],
   )
-  const [safeOverviews] = useSafeOverviews(multiChainGroupSafes)
+  const deployedSafes = useMemo(
+    () => multiChainGroupSafes.filter((safe) => undeployedSafes[safe.chainId]?.[safe.address] === undefined),
+    [multiChainGroupSafes, undeployedSafes],
+  )
+  const { data: safeOverviews } = useGetMultipleSafeOverviewsQuery({ safes: deployedSafes, currency })
```

Comment:
Do we need to pass the walletAddress here as well?

Reply:
I do not pass it to have less dependencies and more likely cache hits. 
This component is only interested in the signers and the threshold. The wallet address fetches the pending actions which is not needed here.